test/error/ErrorTest.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import assert from "assert";
+import { MatterJsErrorCodes, MatterJsError, isMatterJsError } from "../../src/error/MatterJsError";
+
+export function assertMatterJsError(e: unknown): asserts e is MatterJsError {
+    if (!isMatterJsError(e)) {
+        throw e;
+    }
+}
+
+describe("Errors", () => {
+
+    context("Test Errors", () => {
+        it("error with code", () => {
+            try {
+                throw new MatterJsError("test", MatterJsErrorCodes.FabricNotFound);
+            } catch (error) {
+                assertMatterJsError(error);
+                assert.equal(isMatterJsError(error), true);
```

Comment:
Unnecessary

Reply:
kk, will remove but yes already dfone by the other assert